Module not found: Default condition should be last one

Hello,

I am trying to use the library with React and Next `npx create-next-app@latest`

and I get
```
Failed to compile

./app/page.tsx:2:0
Module not found: Default condition should be last one
  1 | import Image from "next/image";
> 2 | import { RTCPeerConnection } from 'werift';
  3 |
  4 | export default function Home() {
  5 | 	const pc = new RTCPeerConnection({});

https://nextjs.org/docs/messages/module-not-found
```
After some digging around I found a solution that worked for me. 
I went into the `node_modules/werift/pacκage.json` and changed the order of the exports like this
```
  "exports": {
    ".": {
      "types": "./lib/webrtc/src/index.d.ts", <------ Moved types first.
      "default": "./lib/webrtc/src/index.js" <------ Default second.
    },
    "./nonstandard": {
      "types": "./lib/webrtc/src/nonstandard/index.d.ts", <----- Same for non standard exports.
      "default": "./lib/webrtc/src/nonstandard/index.js"
    }
  },
  ```
and it seems to work. I am quite novice with typecript, I don't know if this happens to anyone else but I leave it here for others who may face the same issue.

(feel free to close the ticket if you deem staying open doesn't help anyone)

--- a/packages/rtp/package.json
+++ b/packages/rtp/package.json
@@ -13,12 +13,12 @@
   },
   "exports": {
     ".": {
-      "default": "./lib/rtp/src/index.js",
-      "types": "./lib/rtp/src/index.d.ts"
+      "types": "./lib/rtp/src/index.d.ts",
+      "default": "./lib/rtp/src/index.js"
     },
     "./extra": {
-      "default": "./lib/rtp/src/extra/index.js",
-      "types": "./lib/rtp/src/extra/index.d.ts"
+      "types": "./lib/rtp/src/extra/index.d.ts",
+      "default": "./lib/rtp/src/extra/index.js"
     }
   },
   "main": "lib/rtp/src/index.js",

--- a/packages/webrtc/package.json
+++ b/packages/webrtc/package.json
@@ -14,12 +14,12 @@
   },
   "exports": {
     ".": {
-      "default": "./lib/webrtc/src/index.js",
-      "types": "./lib/webrtc/src/index.d.ts"
+      "types": "./lib/webrtc/src/index.d.ts",     
+       "default": "./lib/webrtc/src/index.js"
     },
     "./nonstandard": {
-      "default": "./lib/webrtc/src/nonstandard/index.js",
-      "types": "./lib/webrtc/src/nonstandard/index.d.ts"
+      "types": "./lib/webrtc/src/nonstandard/index.d.ts",
+      "default": "./lib/webrtc/src/nonstandard/index.js"
     }
   },
   "main": "lib/webrtc/src/index.js",

